examples/phi2/phi2.py

```diff
@@ -14,12 +14,12 @@
 
 tokens = tokenizer.encode(prompt)
 
-params=og.SearchParams(model)
+params=og.GeneratorParams(model)
 params.max_length = 200
 params.input_ids = tokens
 
 start_time=time.time()
-output_tokens=model.generate(params)
+output_tokens=model.generate(params)[0]
```

Comment:
does a similar change need to be made in llama.py?

Reply:
I made the samples consistent